mkosi

```diff
@@ -22,6 +22,9 @@ from enum import Enum
 
 __version__ = '1'
 
+if sys.version_info[1] < 5:
```

Comment:
well, we specify "python3" in the shebang header, hence the major is already checked, i only checked the minor...

I am not sure I fully grok tuple comparisons in python3, for example, is (3, 5, 0) < (3, 5) or is (3, 5) < (3, 5, 0)? I am not keen on using expressions whose full effect I really don't grok...